docs/source/conf.py

```diff
@@ -48,17 +48,17 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'Model Mommy'
+project = u'Model Bakery'
```

Comment:
Python 3 FTW! `u` is not needed anymore!

Reply:
Fixed by ae03c23